.github/workflows/e2e-test.yml

```diff
@@ -36,7 +39,7 @@ jobs:
       - name: Install R package dependencies
         uses: r-lib/actions/setup-r-dependencies@v2
         with:
-          extra-packages: local::.
+          packages: Appsilon/rhino@${{ github.sha }}

```

Comment:
Since I am only checking the `tests/e2e` directory I need to install the rhino version of this commit.